ibis/backends/pandas/execution/join.py

```diff
@@ -32,27 +31,14 @@ def execute_cross_join(op, left, right, **kwargs):
     by cross join.
 
     """
-    # generate a unique name for the temporary join key
-    key = f"cross_join_{ibis.util.guid()}"
-    join_key = {key: True}
-    new_left = left.assign(**join_key)
-    new_right = right.assign(**join_key)
-
-    # inner/outer doesn't matter because every row matches every other row
-    result = pd.merge(
-        new_left,
-        new_right,
-        how='inner',
-        on=key,
+    return pd.merge(
+        left,
+        right,
+        how='cross',
```

Comment:
hmm this requires IIRC pandas 1.3.x or great (for the cross)

Reply:
Thoughts on bumping the minimum pandas version?

Reply:
oh actually this is in 1.2. yeah ok with bumping.

Reply:
We're already on 1.2.5, so this is a no-op! Woohoo!

Reply:
@jreback Is this addressed? Can you remove the change request when you get a moment?